app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_in = command.split(" ")[1]
+    file_out = command.split(" ")[2]
```

Comment:
Don't split `command` twice. Create a separate variable for storing result of `command.split()` and use it where you need.